frontend/packages/console-app/src/components/cloud-shell/CloudShellTerminal.scss

```diff
@@ -0,0 +1,8 @@
+.odc-cloudshell-terminal {
+  &__container {
+    background-color: var(--pf-global--BackgroundColor--dark-100);
```

Comment:
This background color is incorrect. We need pure black.

Reply:
So back to `black-10000`?

Reply:
The default terminal theme background color is pure black. So whatever gets us to pure black.
Unless we choose to override. But at this point being consistent with the other terminal is probably best.

Reply:
the black-1000 is the only match for the xterm color